### PR TITLE
Add carthage docs & badge

### LINKED ISSUE #184: Document Carthage setup with EarlGrey gem & Add Carthage badge

### PR DESCRIPTION
**Pull request by bootstraponline**
Fix #184

### CODE REVIEW

**Comment by tirodkar** [comment]
This is great!! Thanks a lot @bootstraponline. LGTM.

File: docs/install-and-run.md [hunk 1]
```diff
@@ -97,6 +97,44 @@ After you successfully run the `pod install` command, open the generated workspa
 
   <img src="images/image02.png" width="250">
 
+### Carthage Installation
+
+EarlGrey supports Carthage via the EarlGrey gem.
```
Comment (khandpur) [inline docs/install-and-run.md]: could we add a link to the gem?

File: docs/install-and-run.md [hunk 2]
```diff
@@ -97,6 +97,44 @@ After you successfully run the `pod install` command, open the generated workspa
...
+
+#### Step 1: Set up a test target
+
+See Step 1 from the `CocoaPods installation` detailed above.
```
Comment (khandpur) [inline docs/install-and-run.md]: could we link to the cocoapods installation section?

File: docs/install-and-run.md [hunk 3]
```diff
@@ -97,6 +97,44 @@ After you successfully run the `pod install` command, open the generated workspa
...
+
+#### Step 2: Configure carthage
+
+Install carthage
+
+- `brew update; brew install carthage`
```
Comment (khandpur) [inline docs/install-and-run.md]: did you mean to add this as a bullet. rest of them are not.

**Comment by bootstraponline** [comment]
Updated.

**Comment by khandpur** [comment]
LGTM

**Pull request merged**